ground/gcs/src/plugins/config/configplugin.cpp

```diff
@@ -75,9 +75,9 @@ bool ConfigPlugin::initialize(const QStringList &args, QString *errMsg)
     connect(telMngr, &TelemetryManager::connected, this, &ConfigPlugin::onAutopilotConnect);
     connect(telMngr, &TelemetryManager::disconnected, this, &ConfigPlugin::onAutopilotDisconnect);
 
-    // And check whether by any chance we are not already connected
```

Comment:
So we're positive that initialization of all plugins (and all connections made) before autopilot connection?  I know it's generally slow/takes a few seconds.. I'm just curious if this is guaranteed by design.